app/test_main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+import pytest
+
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.fixture()
+def mock_url():
+    with mock.patch("app.main.valid_google_url") as mocked_url:
+        yield mocked_url
+
+
+@pytest.fixture()
+def mock_connection():
+    with mock.patch("app.main.has_internet_connection") as mocked_connection:
+        yield mocked_connection
+
+
+def test_return_true_when_url_exists_and_datetime(
+        mock_url,
+        mock_connection):
```

Comment:
it is a bad code style to have brackets at different levels
While creating a dictionary/list - the braces must be located in one of two options: open and start with the text or have a line break between the text.